docs/guide/signing-data.md

```diff
@@ -7,7 +7,7 @@ Since MetaMask makes cryptographic keys available to each user, websites can use
 
 ## Signing Data with MetaMask
 
-If you’d like to jump to some working signature examples, [you can visit this repository](https://github.com/danfinlay/js-eth-personal-sign-examples).
+If you’d like to jump to some working signature examples, [you can visit this repository](https://github.com/MetaMask/test-dapp).
 
```

Comment:
I'm having technical problems with my account

Reply:
Will u be able to add me on again, I just took long to comment

Reply:
> I'm having technical problems with my account 

Checked

Reply:
Thank you

Reply:
Please do not comment on PRs about unrelated topics. If you are looking for support, see: https://metamask.zendesk.com/hc/en-us